veracruz-utils/src/policy/policy.rs

```diff
@@ -40,14 +40,18 @@ use crate::{
         },
     },
 };
+use ring;
```

Comment:
I know this matches a lot of the code in the repo, but moving forward, I've been trying to get the `use` statements as specific as possible and not relying on the whole `rings::digest::...` pattern as much.